### DISCUSSION TITLE
FLAC audio

### DISCUSSION THREAD

**Discussion by xpollen8** [xpollen8/jazzbutcher.com#62] [Announcements]
After thinking about this... And because I'm uploading my first FLAC set right now.

Let's do this for FLAC bundles:

* Create an mp3 from the individual files `sox *.flac [DATE]_JazzButcher_[CITY]_FLAC.mp3`
* Upload the mp3
* Upload the original zipfile

This'll give us the best of both worlds: something to listen to, AND the originals for someone who wants 'em.

**Comment by wnerini** [comment]
LOL, was finally (been a crazy last 10 days) uploading the shows I've gotten, and just saw this. Will merge to MP3 and zip originals...
